eq-author/src/middleware/headers/authHeader.js

```diff
@@ -1,3 +1,19 @@
+import jwt from "jsonwebtoken";
+import config from "config";
+
+const url = `https://securetoken.googleapis.com/v1/token?key=${config.REACT_APP_FIREBASE_API_KEY}`;
```

Comment:
Should this be an environment variable?

Reply:
Thought about it but didn't seem worth adding it too all the pipeline stuff as it's not private and it's unlikely to be reused or re-written at any point. Could go either way though, thoughts?

Reply:
Like you said, it's public URL and I think there's similar URL in the API pointing at google's public keys so I'd say don't worry about it for now. But as soon as the URL needs to change it makes sense to extract it to avoid code changes.